libsolidity/inlineasm/AsmAnalysis.cpp

```diff
@@ -533,19 +533,33 @@ void AsmAnalyzer::expectValidType(string const& type, SourceLocation const& _loc
 
 void AsmAnalyzer::warnOnInstructions(solidity::Instruction _instr, SourceLocation const& _location)
 {
-	static set<solidity::Instruction> futureInstructions{
-		solidity::Instruction::CREATE2,
-		solidity::Instruction::RETURNDATACOPY,
-		solidity::Instruction::RETURNDATASIZE,
-		solidity::Instruction::STATICCALL
-	};
-	if (futureInstructions.count(_instr))
+	// We assume that returndatacopy, returndatasize and staticcall are either all available
+	// or all not available.
+	solAssert(m_evmVersion.hasReturndatacopy() == m_evmVersion.hasStaticCall(), "");
```

Comment:
Does `hasReturndatacopy` also means `hasReturndatasize`?

Reply:
Yes, I can rename it to `hasReturndatacopyAndReturndatasize` or add a comment.

Reply:
I think a comment is enough. Maybe `hasReturndata` was the generic name of the feature?

Reply:
I will change it to `supportsReturndata`.